lib/rust/lexer/definition/src/token.rs

```diff
@@ -24,6 +24,11 @@ pub struct Token {
 }
 
 impl Token {
+    /// Create a new token.
```

Comment:
use shorter docs and dont repeat name of the struct (in case of a rename its easier to keep it up to date). Just write "Constructor." as this is as expressive and informative as your current comment